desktop-src/w8cookbook/operating-system-version-changes-in-windows-8-1.md

```diff
@@ -10,116 +10,93 @@ ms.date: 05/31/2018
 
```

Comment:
This documentation page is substantially duplicated in 2 places:
- https://docs.microsoft.com/en-us/windows/win32/w8cookbook/operating-system-version-changes-in-windows-8-1 (this file)
- https://docs.microsoft.com/en-us/windows/compatibility/operating-system-version-changes-in-windows-8-1

Unfortunately, the other copy of the source file isn't open-source on GitHub as far as I can tell, though I do have internal access to it. I'll submit an internal PR to update it, but the 2 files should really be merged or one deleted altogether.